ow_power_system/src/PrognoserInputHandler.cpp

```diff
@@ -132,6 +133,24 @@ void PrognoserInputHandler::applyValueMods(double& power,
   m_temperature_modifier = 0.0;
 }
 
+
+double PrognoserInputHandler::electricalPower () const
+{
+  return
+    // Power consumption of joints (arm, antenna)
+    m_mechanical_power_to_be_processed / m_efficiency +
+    // Power consumption of other systems (more to be added)
+    electricalPowerLights();
```

Comment:
I found this multi-line return hard to read, can we do this intsead?
 ```suggestion
  return (
    // Power consumption of joints (arm, antenna)
    m_mechanical_power_to_be_processed / m_efficiency +
    // Power consumption of other systems (more to be added)
    electricalPowerLights()
  )
```